cluster/addons/podsecuritypolicies/restricted.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: extensions/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: restricted
+  annotations:
+    kubernetes.io/description: 'restricted allows minimal access for create pods to work.'
+  labels:
+    addonmanager.kubernetes.io/mode: EnsureExists
+spec:
+spec:
+  privileged: false
```

Comment:
hostNetwork, hostPID, hostIPC are all implicitly false, but listing them out makes it easier to understand what this PSP is restricting